File: CS/Form1.cs
using System;
using System.Collections.Generic;
using System.Linq;
using DevExpress.XtraPrinting.Localization;
using DevExpress.XtraPrinting.Preview;
using DevExpress.XtraReports.UI;

namespace WindowsFormsApplication3 {
    public partial class Form1 : DevExpress.XtraEditors.XtraForm {
        public Form1() {
            InitializeComponent();
        }
        private void simpleButton1_Click(object sender, EventArgs e) {
            XtraReport1 report = new XtraReport1();
            report.CreateDocument();
            ReportPrintTool tool = new ReportPrintTool(report);
            tool.PreviewForm.Shown += PreviewForm_Shown;
            tool.ShowPreview();
        }

        private void PreviewForm_Shown(object sender, EventArgs e) {
            SetParametersPanelWidth(sender);
        }
        private  void SetParametersPanelWidth(object sender) {
            PrintPreviewFormEx previewForm = sender as PrintPreviewFormEx;
            DevExpress.XtraBars.Docking.DockPanel parametersPanel = GetParametersPanel(previewForm);
            SetPanelWidth(parametersPanel, 500);
        }
        private  void SetPanelWidth(DevExpress.XtraBars.Docking.DockPanel parametersPanel, int customParamsPanelWidth) {
            parametersPanel.Width = customParamsPanelWidth;
        }
        private  DevExpress.XtraBars.Docking.DockPanel GetParametersPanel(PrintPreviewFormEx previewForm) {
            return previewForm.PrintControl.DockManager.Panels.FirstOrDefault(x => x.Text == GetParametersPanelText());
        }
        private System.String GetParametersPanelText() {
            return PreviewLocalizer.GetString(PreviewStringId.ParametersRequest_Caption);
        }
    }
}

File: VB/Form1.vb
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports DevExpress.XtraPrinting.Localization
Imports DevExpress.XtraPrinting.Preview
Imports DevExpress.XtraReports.UI

Namespace WindowsFormsApplication3
    Partial Public Class Form1
        Inherits DevExpress.XtraEditors.XtraForm

        Public Sub New()
            InitializeComponent()
        End Sub
        Private Sub simpleButton1_Click(ByVal sender As Object, ByVal e As EventArgs) Handles simpleButton1.Click
            Dim report As New XtraReport1()
            Dim tool As New ReportPrintTool(report)
            AddHandler tool.PreviewRibbonForm.Shown, AddressOf PreviewForm_Shown
            tool.ShowRibbonPreview()
        End Sub
        Private Sub PreviewForm_Shown(ByVal sender As Object, ByVal e As EventArgs)
            SetParametersPanelWidth(sender)
        End Sub
        Private Sub SetParametersPanelWidth(ByVal sender As Object)
            Dim previewForm As PrintPreviewRibbonFormEx = TryCast(sender, PrintPreviewRibbonFormEx)
            CType(previewForm.PrintControl.DocumentSource, XtraReport).CreateDocument()
            Dim parametersPanel As DevExpress.XtraBars.Docking.DockPanel = GetParametersPanel(previewForm)
            parametersPanel.Width = 700
        End Sub
        Private Function GetParametersPanel(ByVal previewForm As IPrintPreviewForm) As DevExpress.XtraBars.Docking.DockPanel
            Return previewForm.PrintControl.GetDockPanel(PreviewDockPanelKind.Parameters)
        End Function
    End Class
End Namespace
